scripts/src/main/resources/scripts/command/tomcat

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+TOMCAT_HOME="${DEVON_IDE_HOME}/software/tomcat"
+TOOL_VERSION_COMMAND="tomcat version"
```

Comment:
Indeed the argument to give is `version` (not `-v` or `--version` what would be POSIX standard) so you figured this correctly. However, there is not such command `tomcat`. This is called `catalina` instead. Even worse, for linux/mac it is `catalina.sh`. We could always run this but then we have to test if that still works if all is called from a Windows `CMD` shell. Otherwise, you would need some OS switch like
```
TOMCAT="catalina.sh"
if doIsWindows
then
  TOMCAT="catalina.bat"
fi
TOOL_VERSION_COMMAND="${TOMCAT} version"
```